### PR TITLE
Use default keyword as the main

### PR DESCRIPTION
**Pull request by guybedford**
As discussed in #16, I think this polishes the aesthetics of this proposal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Here’s a complete `package.json` example, for a hypothetical module named `@mo
 
 Within the `"exports"` object, the keys are concatenated on the end of the name field, e.g. `import utc from '@momentjs/moment/timezones/utc'` is formed from `'@momentjs/moment'` + `'/timezones/utc'`.
 
-The main entrypoint is therefore the empty string, `"": "./src/moment.mjs"`. For modules that desire to export *only* a single entrypoint, e.g. `import request from 'request'`, the `"exports"` key itself can be set to the entrypoint:
+The main entrypoint is therefore the empty string, `"default": "./src/moment.mjs"`. For modules that desire to export *only* a single entrypoint, e.g. `import request from 'request'`, the `"exports"` key itself can be set to the entrypoint:
```
Comment (GeoffreyBooth) [inline README.md]: Shouldn’t still say “empty string” here.

**Review by styfle — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Comment by hybrist** [comment]
One thing that was brought up in another thread: If we switch `"/path"` to `"./path"`, the key for the default file could also be `"."`. Would that change things? Or would we still want a keyword?

**Comment by guybedford** [comment]
"." is definitely nicer than "".

@MylesBorins what are your thoughts here?

**Comment by GeoffreyBooth** [comment]
> One thing that was brought up in another thread: If we switch `"/path"` to `"./path"`, the key for the default file could also be `"."`. Would that change things? Or would we still want a keyword?

But then what would the docs say? The left hand side is no longer string concatenation.

Another option is to just split this all up into two keywords:

```js
"nodejs": {
  "exportsMain": "./index.mjs",
  "exports": {
    "/stub": "./dist/stub.mjs"
  }
}

**Comment by hybrist** [comment]
The docs would say "the LHS needs to start with a dot which will be replaced with the package name". I think that's almost as simple as pure concat.

**Comment by GeoffreyBooth** [comment]
> the LHS needs to start with a dot which will be replaced with the package name

Okay, I can get behind that.

If we need to add a separate “mode” field anyway though, should we consider just keeping the main and deep exports as separate properties? Or wrap it all under a `nodejs.module` property or something?

**Comment by guybedford** [comment]
Superseded by #20.

**Pull request closed without merging**